submitter/src/main/kotlin/ac/uk/ebi/biostd/submission/exceptions/InvalidPatternException.kt

```diff
@@ -0,0 +1,27 @@
+package ac.uk.ebi.biostd.submission.exceptions
+
+import java.util.regex.Pattern
+
+/**
+ * Generated when submission is trying to be attached to not existing project.
```

Comment:
Typo: ... be attached to a nonexisting project.

Reply:
this one is not clear for me